### DISCUSSION TITLE
PNY GEFORCE RTX 4070 Ti SUPER 16GB VERTO Overclocked Edition

### DISCUSSION THREAD

**Discussion by osy** [osy/XG_Mobile_Station#5] [Show and tell]
I learned the PCIe 4.0 does work apparently. I guess the last board's routing had some signal integrity issues that was resolved with the new SMD connector that I used on the rev 4 boards.

<img width="291" alt="Screenshot 2024-07-27 150035" src="https://github.com/user-attachments/assets/5365425d-9f5a-4c6a-9671-5d86b8cb03ce">
<img width="691" alt="Screenshot 2024-07-27 145826" src="https://github.com/user-attachments/assets/5fd99eb8-5bda-4b3a-a3ab-ba1e7e99ad41">
<img width="691" alt="Screenshot 2024-07-27 145855" src="https://github.com/user-attachments/assets/28cd7a1c-ab3d-4781-a97f-0d6d2779f36c">

Bought this on sale last week for $819.99 - $90.20 (11% Alienware coupon) - $155 (AmEx offers) - $40.13 (cashback) - $23.95 (credit card cashback) = $510.71. This was the fastest card that will fit the 300W power limit of the XG Station Pro power supply (285W TGP).

**Comment by qqQqq0012** [comment]
Would u mind take a picture for the egpu?

**Reply by osy** [reply]
https://github.com/osy/XG_Mobile_Station/blob/main/Docs/Diary.md#final-build
